docs/spec/README.md

```diff
@@ -1,17 +1,20 @@
-# Introduction
+Each page in the K8TRE Specification contains a specification statement, some background motivation if applicable, illustrative compliance information for a selection of Kubernetes-based TREs, and Q & A. They have been derived from [k8tre GitHub Org Discussions](https://github.com/orgs/k8tre/discussions), as well as discussion within the K8TRE working group.
 
-Each page in this folder contains a K8TRE Specification statement pertaining to a particular topic within the K8TRE Specification. They have been derived from [k8tre GitHub Org Discussions](https://github.com/orgs/k8tre/discussions), as well as discussion within the K8TRE working group, and reviewed by GitHub Pull Request for publication to this site.
+References illustrative Kubernetes-based TREs include:
+- the K8TRE Reference Implementation itself 
+- [TREu](https://www.ucl.ac.uk/advanced-research-computing/sensitive-data-and-trusted-research-environments) – the codebase for the UCL ARC TRE
+- [FRIDGE](https://dareuk.org.uk/how-we-work/ongoing-activities/dare-uk-early-adopters/fridge/)
 
-There a few important clarifications regarding the interpretation of these Specification statements:
+## Interpretation of Specification Statements
 
-1. **Scope** - The K8TRE Specification applies to Kubernetes-based TRE **codebases**. It is written for an audience of K8TRE codebase developers and defines what a K8TRE-compliant codebase must provide. This includes the K8TRE Reference Implementation as well as other codebases such as the UCL ARC TRE and the FRIDGE TRE.
+1. **Scope** - The K8TRE Specification applies to Kubernetes-based TRE *codebases*. It is written for an audience of K8TRE codebase developers and defines what a K8TRE-compliant codebase Must, Should or May provide.
 
-2. **Technology Neutrality** - Beyond its explicit reliance on Kubernetes, the specification is technology-agnostic. However, it may prescribe in detail the capabilities that must be provided by a codebase.
+2. **Technology Neutrality** - Beyond its explicit reliance on Kubernetes, the specification is technology-agnostic. However, it may prescribe capabilities that must be provided by a codebase.
 
-3. **Applicability** – These statements are directed at those developing K8TRE-compliant codebases. They do not apply to administrators or operators running a TRE built on such a codebase (a "deployment").
+3. **Applicability** – These statements are directed at those developing K8TRE-compliant codebases. They are less useful to administrators or operators running a TRE built on such a codebase (a "deployment"). Instead, the TRE codebase being deployed should provide all required documentation.
```

Comment:
Should we add something about this being scoped to "implementing the technical/compute infrastructure of a TRE"? E.g. "This should be read in conjunction with the [SATRE specification](https://satre-specification.readthedocs.io/) and other relevant accreditations."?

Reply:
Have edited the first (scope) bullet.